### ISSUE TITLE
Two Handed Animation

### ISSUE THREAD

**Issue by Moorio69** [HolographicWings/LethalSDK-Unity-Project#50]
I think I've found the reason why this setting doesn't work. I tried a bunch of stuff in the item properties in the Unity Explorer mod and it seems that Item.grabAnim has to be set to "HoldLung" for it to work. When I apply it to one of my items the hand positions are the same as on vanilla items, such as large axle and v-type engine. Hopefully this is the issue and can be fixed easily
![Skärmbild 2024-01-09 000738](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/2d4ae73c-c03b-45de-81b7-ef9b9c0eb947)

**Comment by HolographicWings** [comment]
> I think I've found the reason why this setting doesn't work. I tried a bunch of stuff in the item properties in the Unity Explorer mod and it seems that Item.grabAnim has to be set to "HoldLung" for it to work. When I apply it to one of my items the hand positions are the same as on vanilla items, such as large axle and v-type engine. Hopefully this is the issue and can be fixed easily
> ![Skärmbild 2024-01-09 000738](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/2d4ae73c-c03b-45de-81b7-ef9b9c0eb947)
> 

Oooh neat ! Thanks you for finding this
I'll fix it today

**Comment by Moorio69** [comment]
It still doesn't work. First of all, Item.grabAnim doesn't get set to HoldLung. 
I believe that it is this setting that should set the animation:
![Skärmbild 2024-01-09 190747](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/8a089670-f259-4852-8a16-dbcd02400b0b)
But that doesn't seem to do anything.

And I also think that the Two Handed Animation bool has to be there as well since it has to be set to true in-game for it to work
![Skärmbild 2024-01-09 191901](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/fbbd16d0-d79d-4d7c-994a-b7e483b5b80a)

Hope this fixes it!

**Comment by fredagmys4** [comment]
> > I think I've found the reason why this setting doesn't work. I tried a bunch of stuff in the item properties in the Unity Explorer mod and it seems that Item.grabAnim has to be set to "HoldLung" for it to work. When I apply it to one of my items the hand positions are the same as on vanilla items, such as large axle and v-type engine. Hopefully this is the issue and can be fixed easily
> > ![Skärmbild 2024-01-09 000738](https://private-user-images.githubusercontent.com/33062321/295059218-2d4ae73c-c03b-45de-81b7-ef9b9c0eb947.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ4MzY4OTcsIm5iZiI6MTcwNDgzNjU5NywicGF0aCI6Ii8zMzA2MjMyMS8yOTUwNTkyMTgtMmQ0YWU3M2MtYzAzYi00NWRlLTgxYjctZWY5YjljMGViOTQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA5VDIxNDMxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1N2FkNmYxYmY2ZDAzOWFmOWVkNWNhNDEzYWViYWM3YjEwMzRjZDI2NTc3NDI5NDgyY2MyY2I3OThjN2IzZDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1JW4QqbOL3-VW8Pnms4pYUPXt3pS-PM6v9rUfDsicXQ)
> 
> Oooh neat ! Thanks you for finding this I'll fix it today

It still doesn't work. First of all, Item.grabAnim doesn't get set to HoldLung.
I believe that it is this setting that should set the animation:
![Skärmbild 2024-01-09 190747](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/594fde73-4ae7-458f-8dae-96aef798487d)

But that doesn't seem to do anything.

And I also think that the Two Handed Animation bool has to be there as well since it has to be set to true in-game for it to work
![Skärmbild 2024-01-09 191901](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/478b0bf8-bbe1-4247-a257-d516a558bf77)


Hope this fixes it!

**Comment by HolographicWings** [comment]
@fredagmys4 
It's supposed to set the proper anime string
When I try on my side it works Oo
Are you sure you saved the changes as well

**Comment by fredagmys4** [comment]
> @fredagmys4 It's supposed to set the proper anim string and the boolean When I try on my side it works Oo Are you sure you saved the changes as well

Hmm it still doesn't work...
Are these the correct settings or maybe I am missing something?
![Skärmbild 2024-01-12 185056](https://github.com/HolographicWings/LethalSDK-Unity-Project/assets/33062321/c35ede1d-6639-4751-8614-a8f76123a8e0)

**Comment by HolographicWings** [comment]
> > @fredagmys4 It's supposed to set the proper anim string and the boolean When I try on my side it works Oo Are you sure you saved the changes as well
> 
> Hmm it still doesn't work... Are these the correct settings or maybe I am missing something? ![Skärmbild 2024-01-12 185056](https://private-user-images.githubusercontent.com/33062321/296321019-c35ede1d-6639-4751-8614-a8f76123a8e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODUzNjgsIm5iZiI6MTcwNTA4NTA2OCwicGF0aCI6Ii8zMzA2MjMyMS8yOTYzMjEwMTktYzM1ZWRlMWQtNjYzOS00NzUxLTg2MTQtYThmNzYxMjNhOGUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE4NDQyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1OWI1OTFkZjRiOTZiYjIyZWFiZmUwZmUzZmQ4YWVmMjFiOWI1MjI3ODI4ZTM0ODk4ZTRhNTM2ZWVjMWU5ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Zh5iWIOo2TeoBXwPqNyOk3F66s9c2w-sjULptBiFYQ0)

do you use the last version of LE ? (1.3.18)

**Comment by fredagmys4** [comment]
> do you use the last version of LE ? (1.3.18)
No, totally forgot that was needed. It works now after updating it. Sorry to bother lmao😘
